crates/astria-cli/src/sequencer/ics20_withdrawal.rs

```diff
@@ -0,0 +1,116 @@
+use astria_core::{
+    primitive::v1::{
+        asset,
+        Address,
+    },
+    protocol::transaction::v1alpha1::{
+        action::Ics20Withdrawal,
+        Action,
+    },
+};
+use clap::ArgAction;
+use color_eyre::eyre::{
+    self,
+    WrapErr as _,
+};
+use ibc_types::core::{
+    channel::ChannelId,
+    client::Height,
+};
+
+use crate::utils::submit_transaction;
+
+fn now_plus_5_minutes() -> u64 {
+    use std::time::Duration;
+    tendermint::Time::now()
+        .checked_add(Duration::from_secs(300))
+        .expect("adding 5 minutes to the current time should never fail")
+        .unix_timestamp_nanos()
+        .try_into()
+        .expect("timestamp must be positive, so this conversion would only fail if negative")
+}
+
+#[derive(clap::Args, Debug)]
+pub(super) struct Command {
+    #[arg(long)]
+    pub(crate) amount: u128,
+    #[arg(long)]
+    pub(crate) destination_chain_address: String,
+    /// The source channel used for withdrawal
+    #[arg(long)]
+    pub(crate) source_channel: String,
+    /// The address to refund on timeout, if unset refunds the signer
+    #[arg(long)]
+    pub(crate) return_address: Address,
+    /// A memo to send with transaction
+    #[arg(long, default_value = "")]
+    pub(crate) memo: String,
+    /// The bridge account to transfer from
+    #[arg(long, default_value = None)]
+    pub(crate) bridge_address: Option<Address>,
+    #[arg(long, action(ArgAction::SetTrue))]
+    pub(crate) use_compact: bool,
```

Comment:
What is compact?

Reply:
should have been "compat" for the compatibility addresses. this has been fixed.